python/.konan

```diff
@@ -0,0 +1,24 @@
+# This file contains some configurations for your konan deployment
+# Read the comments carefully, and replace any string of the format {tempalte-value} with your actual value
+
+# Section related to where to push your generated image to
+# [docker-registry]
+
+# Possible values include: docker.io, quai.io, registry.gitlab.com and konancr.azurecr.io
+KONAN_CONTAINER_REGISTRY=docker.io
```

Comment:
I think it might be better to have this variable be `CONTAINER_REGISTRY`and have `KONAN_CONTAINER_REGISTRY` be a boolean that indicates whether to use KCR or not, if yes, then `CONTAINER_REGISTRY` will be set to `konan.azurecr.io`.

Reply:
The `KONAN_` prefix to the environment variables is meant to have our own namespace and avoid collisions with other environment variables.
The `KONAN_CONTAINER_REGISTRY` variable reads as the "container registry URL the konan scripts will use"

Accordingly, I would rather keep it like this to avoid possible confusion on variable precedence if for example a user would set `KONAN_USE_KONAN_CONTAINER_REGISTRY=True` and `KONAN_CONTAINER_REGISTRY=docker.io`

Reply:
Okay makes sense, maybe make it `KONAN_USE_KCR` for brevity.

Reply:
Will refactor these variables to `KONAN_USE_KCR`, `KONAN_USE_GCR`, `KONAN_USE_ECR`, etc ..., `KONAN_USE_OTHER` later when adding support for `ecr` and `gcr`.
I feel that using the `KONAN_USE_KCR` variable now will be confusing if user is allowed to enter a `KONAN_CONTAINER_REGISTRY` url.